openlibrary/i18n/__init__.py

```diff
@@ -111,7 +112,9 @@ def load_locale(lang):
 class GetText:
     def __call__(self, string, *args, **kwargs):
         """Translate a given string to the language of the current locale."""
-        translations = load_translations(web.ctx.get('lang', 'en'))
+        # Get the website locale from the global ctx.lang variable, set in i18n_loadhook
+        website_locale = web.ctx.lang        
+        translations = load_translations(website_locale)
```

Comment:
Extra variable reduces readability here a bit :) The comment says it all!

```suggestion
        translations = load_translations(web.ctx.lang)
```